server/src/main/java/com/google/speech/tools/voxetta/data/PromptBuilder.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.speech.tools.voxetta.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+
+/**
+ * Builder pattern for Prompts.
+ */
+public class PromptBuilder {
```

Comment:
@everyone--for consistency across the server should the Builder be in a separate file or be contained within the class for which it's building?

Reply:
Up to preference of the team.

Internally we have this thing called Autovalue which generates these classes for us, and that tool puts them nested inside the class.